ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -68,9 +68,19 @@ class FaultInjector
 private:
   float powerTemperatureOverloadValue;
   
```

Comment:
same as next comment

Reply:
this is the power work, so i'll include it in the other PR